draft-ietf-oauth-device-flow.xml

```diff
@@ -636,6 +636,14 @@
           from. Thus, signals from the approving user's session and device are
           not relevant to the trustworthiness of the client device.
           </t>
+	  <t>
+	    Note that if an authorization server used with this flow is malicious,
+	    then it could man-in-the middle the backchannel flow to another authorization server.
+	    For this to be possible, the device manufacturer must either be the attacker
+	    or be using an authorization server that is controlled by an attacker
+	    for this attack to be possible. In part,
+	    the person purchasing the device is counting on it and its business partners to be trustworthy.
+	  </t>
```

Comment:
The one mitigating factor here is that the user would end up on the wrong site when authorizing, and may notice. As John put it: "They would hopefully notice that they are being sent to the wrong site.". Should we include any mention of this?